crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/AppealContributionService.java

```diff
@@ -42,33 +43,33 @@ private AssessmentResult determineAssessmentResult(List<ApiAssessment> assessmen
     public ApiMaatCalculateContributionResponse calculateAppealContribution(
             CalculateContributionDTO calculateContributionDTO) {
         AssessmentResult assessmentResult = determineAssessmentResult(calculateContributionDTO.getAssessments());
+        ApiCrownCourtOutcome latestAppealOutcome = Optional.ofNullable(calculateContributionDTO.getCrownCourtOutcomeList())
+                .orElse(Collections.emptyList())
+                .stream()
+                .filter(outcome -> CrownCourtOutcomeType.APPEAL.getType().equals(outcome.getOutcome().getType()))
+                .reduce((first, second) -> second)
```

Comment:
Can there be multiple appeal outcomes and how do we identify the latest appeal outcome from the list? Do we need to sort the outcomes by date?

Reply:
Looked into this and yes there can be multiple outcomes, the last one in the list is the most recent. Though whilst looking I noticed that previously we were reducing the list and then filtering the appeal type so I've amended the above logic as it would produce a different outcome (get last appeal, rather than check is last outcome an appeal).